<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xview</class>
 <widget class="QMainWindow" name="Xview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_metadata">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_move_all">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Move all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_reset_metadata">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset metadata</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_track">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Track</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="action_open_project">
   <property name="text">
    <string>&amp;Open project</string>
   </property>
  </action>
  <action name="action_save_project">
   <property name="text">
    <string>&amp;Save project</string>
   </property>
  </action>
  <action name="action_save_datasets_as_text">
   <property name="text">
    <string>Save datasets as text</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_merge">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_combine_and_save_as_text">
   <property name="text">
    <string>Combine and save as text</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
